properties/P000058.md

```diff
@@ -1,10 +1,14 @@
 ---
 uid: P000058
-name: Smaller than the continuum
+name: Cardinality $\lt\mathfrak c$
+aliases:
+  - Smaller than the continuum
+  - Cardinality $\lt\beth_0$
 refs:
   - mr: MR2048350
     name: General Topology (Willard)
 ---
-A set $S$ has cardinality < $\mathfrak(c)$ if there is an injective function (not necessarily continuous) $f:S \rightarrow \mathbb{R}$ but no such bijective function.
+
+There exists an injective function from the space into $\mathbb R$, but no such surjective function.
```

Comment:
Can't we just say something more straightforward, like "The cardinality of the space is less than the cardinality of $\mathbb R$"?  Of course it's equivalent, based on Schroeder-Bernstein, etc, but people are supposed to know about the basics of cardinal numbers, and if they don't, they are not going to learn it here.

Reply:
Addressed in afd393d